libs/design-system/tailwind.config.js

```diff
@@ -39,6 +40,20 @@ export default {
       18: '4.5rem', // 72px
       20: '5rem', // 80px
     },
+    colors: {
+      transparent: 'transparent',
+      current: 'currentColor',
+      'ocean-blue': '#00011d',
+      cyan: '#acd8ff',
+      pink: '#fcb2d9',
+      green: '#2ee59d',
+      white: '#ffffff',
+      grey: {
+        light: '#798186',
+        DEFAULT: '#212529',
+      },
+      black: '#000000',
```

Comment:
So finally you went for a semantic naming rather than functional (eg. primary-bg, ...)?

Reply:
Yes. As I went over the code again I finally thought it made more sense like this.